### PR TITLE
Monika "amnesia" greeting unlocked following discussion of Flowers For Algernon book

### PR DESCRIPTION
**Pull request by kaido1224**
Monika "amnesia" topic after discussing flowers for algernon book.

### CODE REVIEW

**Comment by kaido1224** [comment]
Oops. That's what I get for rushing.

**Comment by kaido1224** [comment]
OK it's fixed.

File: Monika After Story/game/script-greetings.rpy [hunk 1]
```diff
@@ -691,3 +691,25 @@ label greeting_sunshine:
     m 1l "I was just singing to myself to pass time."
     m 1b "But now that you're here, we can spend some time together."
     return
+
+if renpy.seen_event("monika_algernon"):
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-greetings.rpy]: oh shit wait `seen_event` is a custom function written by us, not a renpy function. You should use `seen_label` here instead.

**Comment by kaido1224** [comment]
Monik told me to use seen_event over seen_label

**Comment by ashameimaru** [comment]
Question on the programming side: does the [tempname] variable get kept or is it dumped after this is all done? (I'm thinking for purposes of memory cleanup?)

**Comment by kaido1224** [comment]
@Thepotatoguy not sure how to answer that.

**Comment by ThePotatoGuy** [comment]
It's local since it's defined in a `label`. From the looks of the code though, you should probably just avoid using `tempname` and use `fakename` since you have `tempname = fakename` which is redundant in this case.

**Comment by kaido1224** [comment]
Thanks for the input, I went ahead and fixed it to be the way that you suggested. Should be ready for review now.

**Comment by monikLover** [comment]
Closing due to complications. Will be on hold.

**Comment by kaido1224** [comment]
OK, we can reopen this now.

**Pull request closed without merging**